### PR TITLE
Remove Helm Chart in favor for upstream

### LINKED ISSUE #9: Add Upstream Helm Chart
Story
- As a user, I wish to be able to deploy the Backstage Showcase app to a Kubernetes cluster.

Description
- Currently, the Helm Chart that we are utilizing within our repository is the helm-backstage Janus-IDP Helm Chart and is not the correct Helm Chart that we are going to be using to deploy the showcase app with. This can lead to some confusion whenever we start getting more involvement from the community and needs to be address. The correct Helm Chart should instead come from the upstream Backstage repository.

AC
- Upstream Helm Chart added to the repository
- Documentation on how to deploy to a cluster using the Helm Chart.

Additional Notes
- None so far

### PR DESCRIPTION
**Pull request by PatAKnight**
## What does this PR do / why we need it
This PR removes the Backstage Showcase Helm Chart in favor for the Upstream Backstage Helm Chart. The original Helm Chart that we were using came from the helm-backstage repository. We are deciding to center around the Upstream version as that is what the community of Backstage will most likely be using in the future.

I decided against pulling in the Helm Chart from the upstream Backstage repository. My reasoning is that it could become difficult to ensure that our chart stays up to date with the upstream and could lead to some issues in the future. Instead I am proposing the use of the `helm repo add` commands to add the Backstage Helm Chart. The details can be found in the documentation that was added as part of this PR.

## Which issue(s) does this PR fix

Fixes #9 

## PR acceptance criteria

- [ ] Unit Tests
- [ ] E2E Tests
- [x] Documentation

## How to test changes / Special notes to the reviewer
Currently, there is a bug with the Dockerfile where we are unable to deploy to OpenShift. I have detailed the bug in issue #12. This means that to build the image at this time, they will have to go to the [redhat-backstage-build](https://github.com/janus-idp/redhat-backstage-build) repo and copy the Dockerfile over.

### CODE REVIEW

File: docs/helm-chart.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Setting up to use the Helm Chart
+
+The Helm Chart that this project is currently using is the Backstage Helm Chart. This Helm Chart is used to simplify the process of deploying the Backstage Showcase app to a Kubernetes cluster. For me information on the Backstage Helm Chart consult the [official documentation](https://github.com/backstage/charts).
```
Comment (schultzp2020) [inline docs/helm-chart.md:3]: `For me information` -> `For more information`

File: docs/helm-chart.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Getting Started
+
+Ensure that you have Helm installed on your local machine, you can find out more information on the [Helm website](https://helm.sh/docs/intro/install/).
+
+Next, is to add the Backstage Helm Chart and its dependent repository.
```
Comment (schultzp2020) [inline docs/helm-chart.md:9]: The comma isn't necessary.

File: docs/helm-chart.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+## Deploying to a Kubernetes cluster
+
+This guide will have a focus deploying to an OpenShift cluster as that is the main target for this project.
+
+Build and push an image of the Backstage Showcase app that you can test.
+
+```shell
+docker build -t <registry>/<repository>:<tag> .
```
Comment (schultzp2020) [inline docs/helm-chart.md:23]: We should probably switch these commands to podman.

File: docs/helm-chart.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+docker push <registry>/<repository>:<tag>
+```
+
+Create a file to hold environment variables called values-backstage-showcase.yaml and populate it with the following content.
```
Comment (schultzp2020) [inline docs/helm-chart.md:27]: I like to wrap filenames in sentences with backticks. i.e. `values-backstage-showcase.yaml`

File: docs/helm-chart.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+```shell
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add backstage https://backstage.github.io/charts
+```
+
+## Deploying to a Kubernetes cluster
+
+This guide will have a focus deploying to an OpenShift cluster as that is the main target for this project.
+
+Build and push an image of the Backstage Showcase app that you can test.
```
Comment (schultzp2020) [inline docs/helm-chart.md:20]: Should this be an ordered list?

1. Build and push an image of the Backstage Showcase app that you can test.
   ...
2. Create a file to hold environment variables called values-backstage-showcase.yaml and populate it with the following content.
   ...
3. Deploy the Backstage Showcase app to the OpenShift cluster

etc

File: docs/helm-chart.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+Finally forward the port to access the Backstage Showcase app
+
+```shell
+oc port-forward -n backstage svc/backstage 7007:7007
+```
+
+The app can be accessed at http://localhost:7007
+
+## More Information
```
Comment (schultzp2020) [inline docs/helm-chart.md:70]: I would rename this to `Additional Resources`.

**Review by schultzp2020 — CHANGES_REQUESTED** [review]

**Comment by PatAKnight** [comment]
@schultzp2020, thanks for taking a look. I updated everything based on your suggestions.

**Review by schultzp2020 — APPROVED** [review]

**Pull request merged**